### ISSUE TITLE
BaramMesh export never finishes

### ISSUE THREAD

**Issue by lukaville** [nextfoam/baram#81]
**Describe the bug**
BaramMesh export never finishes, there are no errors in the log.

**To Reproduce**
Steps to reproduce the behavior: see video https://i.imgur.com/vnzcdpM.mp4

I have added a mesh from STL file and a Hex6 shape and used all default parameters.

Export doesn't finish even after waiting for a long time.

**Expected behavior**
Export finishes

**Desktop (please complete the following information):**
 - OS: macOS 14.3 (23D56)
 - Version 24.1.2

**Comment by kdg5424** [comment]
Hi @lukaville.
This is Donggyu from NEXTFOAM.
I suspect two things.

First, the location of the point. -I mean the point at the regions step.-
It should be on the fluid region.

And second, the scale of the stl looks very big and large.
So if you decide the default size at the base grid step, the mesh may not be generated properly.
And it could lead the problem for generating mesh.

If it is ok, could you send us the stl file?
After we check it, I will tell you.

Regards

Donggyu

**Comment by lukaville** [comment]
Hi Donggyu, attaching the stl file: https://drive.google.com/file/d/1ULoQ5W5HzAPQ1dE7rHr84WKFKF6LwshR/view?usp=sharing.

> So if you decide the default size at the base grid step, the mesh may not be generated properly.
I have tried to put a reasonable size of the grid for the stl file and the correct region (above the imported mesh and below the boundary) and it still didn't work :( In the video, I used all default parameters just to simplify the reproduction of the issue.

Thanks!
